bin/main

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+# rubocop:disable Style/GlobalVars, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity, Metrics/MethodLength
+require 'colorize'
+require_relative '../lib/board'
+require_relative '../lib/win'
+
+$exit = 0
+$winner = 0
+$player1 = true
+$count = 0
+$board = Board.new
+$current_player = 1
```

Comment:
[OPTIONAL] Try to avoid using global variables in your project. It's considered a bad practice, you can read more about it [here](https://www.tutorialspoint.com/ruby/ruby_variables.htm).